<launch>

  <node pkg="mbf_rrts_planner" type="mbf_rrts_vis_separate_node" name="rrts_planner_A" output="screen" >
    <remap from="map_topic" to="grid_map_filter_A/dummy_filtered_map"/>
    <remap from="planned_path" to="planned_path_A" />
    <remap from="generate_plan" to="generate_plan_A" />
  </node>
  
</launch>